src/test/java/net/logstash/logback/encoder/LogstashEncoderTest.java

```diff
@@ -156,34 +151,34 @@ public void basicsAreIncludedWithShortenedNames() throws Exception {
     public void customDecorators() throws Exception {
         encoder.stop();
         encoder.setJsonFactoryDecorator(new JsonFactoryDecorator() {
-            
+
             @Override
             public JsonFactory decorate(JsonFactory factory) {
                 return factory.disable(JsonGenerator.Feature.QUOTE_FIELD_NAMES);
             }
         });
-        
+
         encoder.setJsonGeneratorDecorator(new JsonGeneratorDecorator() {
-            
+
             @Override
             public JsonGenerator decorate(JsonGenerator generator) {
                 return generator.useDefaultPrettyPrinter();
             }
         });
-        
+
         encoder.start();
         final long timestamp = System.currentTimeMillis();
-        
+
         ILoggingEvent event = mockBasicILoggingEvent(Level.ERROR);
         when(event.getTimeStamp()).thenReturn(timestamp);
-        
+
         byte[] encoded = encoder.encode(event);
-        
+
         String output = new String(encoded, "UTF-8");
-        
+
         assertThat(output).isEqualTo(String.format(
                 "{%n"
-                + "  @timestamp : \"" + FastDateFormat.getInstance("yyyy-MM-dd'T'HH:mm:ss.SSSZZ").format(timestamp) + "\",%n"
+                + "  @timestamp : \"" + String.valueOf(timestamp) + "\",%n"
```

Comment:
Same comment as above. Why is the expected value here a long value converted to a string?

Reply:
I missunderstood the StringTimestampWriter, as a result I've done some wrong changes here